packages/ariakit/src/form/form-reset.ts

```diff
@@ -0,0 +1,81 @@
+import { MouseEvent, useCallback } from "react";
+import { useEventCallback } from "ariakit-utils/hooks";
+import { useStore } from "ariakit-utils/store";
+import {
+  createComponent,
+  createElement,
+  createHook,
+} from "ariakit-utils/system";
+import { As, Props } from "ariakit-utils/types";
+import { ButtonOptions, useButton } from "../button";
+import { FormContext } from "./__utils";
+import { FormState } from "./form-state";
+
+/**
+ * A component hook that returns props that can be passed to `Role` or any other
+ * Ariakit component to render a reset buttom in a form.
+ * @see https://ariakit.org/components/form
+ * @example
+ * ```jsx
+ * const state = useFormState();
+ * const props = useFormReset({ state });
+ * <Form state={state}>
+ *   <Role {...props}>Reset</Role>
+ * </Form>
+ * ```
+ */
+export const useFormReset = createHook<FormResetOptions>(
+  ({ state, ...props }) => {
+    state = useStore(state || FormContext, [`submitting`]);
```

Comment:
Nit:
```suggestion
    state = useStore(state || FormContext, ["submitting"]);
```

Reply:
Ah yeah, I developed this first in my own project and copied it over. I have a different set of eslint rules that enforces backticks. Will change.